packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -372,6 +388,10 @@ class _DraggableState<T> extends State<Draggable<T>> {
           _activeCount -= 1;
           _disposeRecognizerIfInactive();
         }
+        if(mounted && widget.onDragEnd != null)
+          widget.onDragEnd(DraggableDetails._(
+              wasAccepted: wasAccepted, velocity: velocity, offset: offset)
```

Comment:
nit: if the body if the if block spans more than 2 lines, wrap it in braces.
nit: indent by 2 spaces rather than 4. Flutter style would have this wrapped as:

```dart
          widget.onDragEnd(DraggableDetails._(
            wasAccepted: wasAccepted,
            velocity: velocity,
            offset: offset,
          ));
```
...or:
```dart
          widget.onDragEnd(DraggableDetails._(wasAccepted: wasAccepted, velocity: velocity, offset: offset));
```

Reply:
Consider it done.

Reply:
Forgot to add brackets to the if statement block. Doing it now.

Reply:
Added a fix to all nits.